site/annotations-tutorial.md

```diff
@@ -11,37 +11,12 @@ deployment](https://github.com/weaveworks/flux-get-started) and click on the
 
 ## Setup
 
-Get the source code of Flux:
+First, please [install fluxctl](fluxctl.md#installing-fluxctl).
```

Comment:
Can you put these (doc related) changes in a separate commit / PR, so we are able to merge them _after_ the `fluxctl` release has been made?

Reply:
Sure

Reply:
Done!